embulk-core/src/test/ruby/monkey_strptime/mri/date/test_date_strptime.rb

```diff
@@ -1,515 +0,0 @@
-# frozen_string_literal: false
```

Comment:
Tested in https://github.com/embulk/embulk-util-rubytime/blob/v0.3.2/src/jrubyTest/resources/ruby/test/date/test_date_strptime.rb